### DISCUSSION TITLE
Auto hide scrollbar on tabs panel collapse

### DISCUSSION THREAD

**Discussion by zenwav** [goastian/midori-desktop#57] [General]
Currently, the scrollbar stays visible even when the tabs panel is collapsed, leading to unnecessary visual clutter. This is particularly noticeable when using dark mode, as the white scrollbar stands out. It would enhance the user experience if the scrollbar could auto-hide after the panel is collapsed, similar to the functionality in Microsoft Edge

**Comment by Ponchale** [comment]
This has now been resolved and we are working on releasing the update, thank you very much for the report.

**Comment by zenwav** [comment]
Any update on this yet?

**Reply by Ponchale** [reply]
Yes of course de versión 11.4 look the repo.
